dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -0,0 +1,56 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Escort a Dvaered colonel to a system">
+ <unique />
+ <priority>1</priority>
+ <chance>24</chance>
+ <location>Bar</location>
+ <spob>Dvaer Prime</spob>
```

Comment:
In general, we try to avoid limiting missions to a single spob (with some exceptions where it is necessary). If possible, it should be a bit more flexible.